network-api/networkapi/buyersguide/factory.py

```diff
@@ -29,6 +30,14 @@ def get_extended_yes_no_value():
     return choice(options)
 
 
+class BaseProductPrivacyPolicyLinkFactory(DjangoModelFactory):
+    class Meta:
+        model = BaseProductPrivacyPolicyLink
+
+    label = Faker('sentence')
+    url = Faker('url')
+
```

Comment:
the product `product` fields gets presupplied in https://github.com/mozilla/foundation.mozilla.org/pull/4588/files#diff-cb1968534bb82afb1947f850970e06d7R107